### PR TITLE
105642 - Add metric parameter to TrackEvent call

### PR DESCRIPTION
**Pull request by jsoi-eq**
Add metric parameter to TrackEvent call

[AB#105642](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/105642)

### CODE REVIEW

File: src/Equinor.ProCoSys.Common/Telemetry/ApplicationInsightsTelemetryClient.cs [hunk 1]
```diff
@@ -19,8 +19,8 @@ public ApplicationInsightsTelemetryClient(TelemetryConfiguration telemetryConfig
             _ai = new TelemetryClient(telemetryConfiguration);
         }
 
-        public void TrackEvent(string name, Dictionary<string, string> properties)
-            => _ai.TrackEvent(name, properties);
+        public void TrackEvent(string name, Dictionary<string, string> properties, Dictionary<string, double> metrics = null)
```
Comment (TordJoranger) [inline src/Equinor.ProCoSys.Common/Telemetry/ApplicationInsightsTelemetryClient.cs:22]: Is it backwards compatible?

Comment (TordJoranger) [reply src/Equinor.ProCoSys.Common/Telemetry/ApplicationInsightsTelemetryClient.cs:22]: Optional param, so I should think so, but not 100% certain.

**Review by TordJoranger — APPROVED** [review]

**Review by TordJoranger — COMMENTED** [review]
Up version?

**Comment by jsoi-eq** [comment]
> Up version?

How?

**Pull request merged**